runtime/integration-tests/src/relaychain/kusama_cross_chain_transfer.rs

```diff
@@ -115,7 +136,7 @@ fn transfer_sibling_chain_asset() {
 				MultiLocation::new(
 					1,
 					X2(
-						Parachain(2001),
+						Parachain(SIBLING_ID),
```

Comment:
This is NonReserve case, Karura(2000)-[Bifrost2002 token]->Sibling(2001), can we add another testcase: Karura-[Bifrost token]->Bifrost?

Reply:
That would be the same integration test as before - the problem is that Native BNC token is not setup correctly in Karura runtime(it is defined as external token with a different parachain ID, not a native currency). If we want the setup for the test, we will have to add BNC to 
https://github.com/AcalaNetwork/Acala/blob/950a179bcb53e5a84e5f960fdbed34128e3d23d7/runtime/karura/src/xcm_config.rs#L453
which would not fit the Karura runtime.

see how Bifrost setup BNC and how Karura setup KAR
https://github.com/bifrost-finance/bifrost/blob/f862ae756eccb444d5ed0aa8617803c2da94d2e5/runtime/bifrost-kusama/src/xcm_config.rs#L207

Perhaps we can test transfering KARs from Karura to sibling chain? that will fit the current setup

Alternatively I will have to actually bring in Bifrost's Runtime to our integration test, but i think that would be a bit of overkill

Reply:
With the current setup, when Karura sends BNC over to 2001, it's gonna arrive as (parent 0, X1([0, 1)], which we can register as a foreign asset. The only way to change that is to setup the currency correctly (like how bifrost do it here:
https://github.com/bifrost-finance/bifrost/blob/f862ae756eccb444d5ed0aa8617803c2da94d2e5/runtime/bifrost-kusama/src/xcm_config.rs#L207
)

I've added the test, but i dont think there's much benefit we get from it